### PR TITLE
[FEATURE] Github icon in Contributors page

### PR DESCRIPTION
**Pull request by BhartiNagpure**
#537 Github Icon
I'm added github icon in contributors page.

## Pull Request Checklist

Please make sure that your PR meets the following requirements:

- [x] The app is working correctly after applying the changes.
- [x] The app is responsive and displays properly on different screen sizes and devices.
- [x] The changes are thoroughly documented in the appropriate files or documentation sections.
- [x] The coding conventions and style guidelines are followed consistently.

## Checklist

Please check all applicable boxes and provide any additional information if needed.

- [x] I have tested the changes locally and ensured that the app is functioning as expected.
- [x] I have verified that the app is responsive and looks good on various screen sizes and devices.
- [x] I have reviewed the documentation and ensured that it accurately reflects the changes made.
- [x] I have followed the coding conventions and style guidelines defined in the project.
- [x] I have run the automated tests and all of them pass.
- [x] I have considered the performance implications of the changes made.
- [x] I have considered the security implications of the changes made.
- [x] I have checked for any potential regressions or side effects caused by the changes.
- [x] I have performed a self-review of my code.
- [x] Starred the repository.
- [x] Added My Name to the Contributor's List.
- [x] Followed the repository's [Contributing Guidelines](/CONTRIBUTING.md).
- [x] I ran the app and tested it locally to verify that it works as expected.

# Screenshots - If Any (Optional)
Screenshot 2023-07-18 132904

## Related Issues

#537 [FEATURE REQUEST] GitHub Icon in Contributor Page.

## Reviewers

Please tag the relevant team members or reviewers who should review this pull request.

### CODE REVIEW

**Comment by BhartiNagpure** [comment]
@anmode I'm done with my work please review it.

File: styles/Contributor.module.css [hunk 1]
```diff
@@ -33,3 +33,19 @@
...
+  background-color: rgb(45, 44, 44);
+  padding: 7px;
+  border-radius: 10px;
+}
+.githubName{
+  font-size: 20px;
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  margin-left: 8px;
+  color: gainsboro;
+}
```
Comment (Harshil-Jani) [inline styles/Contributor.module.css]: Add new line at End of File

**Comment by Harshil-Jani** [comment]
image

@BhartiNagpure Can you also make this aligning for the mobile views ? It looks misaligned on mobile screens.

**Comment by BhartiNagpure** [comment]
Screenshot 2023-07-18 155642
 @Harshil-Jani alignment for mobile view

**Comment by Harshil-Jani** [comment]
Looks great. Thanks @BhartiNagpure

**Pull request merged**